### PR TITLE
node-monitoring: Use new logging framework to raise alerts

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
@Tendrl/tendrl-core Please review

File: node_monitoring/tendrl/node_monitoring/plugins/handle_collectd_notification.py [hunk 1]
```diff
@@ -64,15 +71,27 @@ def collectd_to_tendrl_alert(collectd_alert, collectd_message):
...
+        "alerting",
+        {
+            'message': json.dumps(tendrl_alert)
+        },
+        node_id=node_context_id
+    )
+    Event(
+        msg,
+        socket_path=config['logging_socket_path']
+    )
```
Comment (shtripat) [inline node_monitoring/tendrl/node_monitoring/plugins/handle_collectd_notification.py]: I would rather code it as below 

```
Event(
    Message (
        "notice",
        tendrl_ns.publisher_id # which is set while main alerting module start
        {"message": json.dumps(tendrl_alert)},
        node_id=tendrl_ns.node_context.node_id
    ),
    socket_path = config['logging_socket_path']
)
```

File: node_monitoring/etc/tendrl/node-monitoring/node-monitoring.conf.yaml.sample [hunk 2]
```diff
@@ -5,3 +5,4 @@ etcd_port: 2379
 etcd_connection: 0.0.0.0
 log_cfg_path: /etc/tendrl/node-monitoring/node-monitoring_logging.yaml
 tendrl_ansible_exec_file: $HOME/.tendrl/node-monitoring/ansible_exec
+logging_socket_path: "/var/run/tendrl/.message.sock."
```
Comment (shtripat) [inline node_monitoring/etc/tendrl/node-monitoring/node-monitoring.conf.yaml.sample]: Name of the socket file should be `/var/run/tendrl/message.sock` as per latest discussions. So change accordingly.

File: node_monitoring/tendrl/node_monitoring/plugins/handle_collectd_notification.py (hunk 1 above)
Comment (anmolbabu) [inline node_monitoring/tendrl/node_monitoring/plugins/handle_collectd_notification.py]: Done

File: node_monitoring/etc/tendrl/node-monitoring/node-monitoring.conf.yaml.sample (hunk 2 above)
Comment (anmolbabu) [inline node_monitoring/etc/tendrl/node-monitoring/node-monitoring.conf.yaml.sample]: Done

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat Please review

File: node_monitoring/etc/tendrl/node-monitoring/node-monitoring.conf.yaml.sample [hunk 3]
```diff
@@ -5,3 +5,4 @@ etcd_port: 2379
 etcd_connection: 0.0.0.0
 log_cfg_path: /etc/tendrl/node-monitoring/node-monitoring_logging.yaml
 tendrl_ansible_exec_file: $HOME/.tendrl/node-monitoring/ansible_exec
+logging_socket_path: /var/run/tendrl/message.sock
```
Comment (r0h4n) [inline node_monitoring/etc/tendrl/node-monitoring/node-monitoring.conf.yaml.sample]: socket name is ".message.sock"

Comment (GowthamShanmugam) [inline node_monitoring/etc/tendrl/node-monitoring/node-monitoring.conf.yaml.sample]: @r0h4n no i gave socket name is message.sock only.

Comment (GowthamShanmugam) [inline node_monitoring/etc/tendrl/node-monitoring/node-monitoring.conf.yaml.sample]: @r0h4n mrugesh told me to change like this because it should visible

**Pull request merged**